install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
 $ /usr/sbin/setsebool -P virt_sandbox_use_samba on
 ----
 
+. Type the `mount` command to check `dir_mode` and `file_mode` permissions, for example:
++
+----
+$ mount
+----
+
+If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
+default value `0755` to `0777` (or `0775`). This manual step is required because
```

Comment:
Do we have an opinion on one or the other, or do we expect the user to know the difference?

Remove the parentheses.